.github/workflows/test.yml

```diff
@@ -20,6 +20,6 @@ jobs:
       run: docker run --rm --name hermez-db-test -p 5432:5432 -e POSTGRES_DB=hermez -e POSTGRES_USER=hermez -e POSTGRES_PASSWORD="${{ secrets.POSTGRES_PASS }}" -d postgres
     - name: Test
       env:
-        POSTGRES_PASS: ${{ secrets.POSTGRES_PASS }}
+        PGPASSWORD: ${{ secrets.POSTGRES_PASS }}
```

Comment:
Did you test it? Does it work?

This env vars is for the Postgres containers. Maybe it can break the test workflow. We are using `POSTGRES PASS` in the docker commands. Check line 20:

https://github.com/hermeznetwork/hermez-node/blob/eff2955a0490d837e56761cbf551e06626f6f622/.github/workflows/test.yml#L20

Reply:
The tests seem to have passed successfully. 
 https://github.com/hermeznetwork/hermez-node/runs/2378696525?check_suite_focus=true

Both make test-unit and make test-api-server work locally as well.

Reply:
POSTGRES_PASS was used only in tests, docker container uses -e POSTGRES_PASSWORD= on the command line.

I decided to rename env var because:
 - new variables are added. The POSTGRES_ prefix is too long, it can be annoying if you need to specify multiple variables on the command line;
 - these variable names are well known and understood by most of Postgres related software (at least all software using libpq)
https://www.postgresql.org/docs/current/libpq-envars.html

Reply:
You are setting the password 2 times, first, one in the command, using the var `POSTGRES_PASSWORD`: 

```
run: docker run --rm --name hermez-db-test -p 5432:5432 -e POSTGRES_DB=hermez -e POSTGRES_USER=hermez -e POSTGRES_PASSWORD="${{ secrets.POSTGRES_PASS }}" -d postgres
```

And the second one by another env var `PGDATABASE`

Reply:
And this is not related to the local unit tests. The `make` command goes to work. You must test it against a GitHub action pipeline. GitHub uses the workflow folder to creates the CI pipeline

Reply:
also, because we are using it `-e POSTGRES_PASSWORD="${{ secrets.POSTGRES_PASS }}"`  we don't need this env var, or we can do `POSTGRES_PASSWORD="$(POSTGRES_PASSWORD)"` and use it

Reply:
or if do you prefer to use the new Postgres variables you can change it to `-e PGPASSWORD="${{ secrets.POSTGRES_PASS }}` along if the others in the command

Reply:
> first, one in the command, using the var `POSTGRES_PASSWORD`:

This variable is for the docker container
https://hub.docker.com/_/postgres
It only exists inside the container

> And the second one by another env var `PGDATABASE`

This variable is used by `go test` (to connect to the docker container)

Reply:
yes, sure, but we are not using this reference, because inside the container was getting the variables directly from the github secret action not from this another var:

`-e POSTGRES_PASSWORD="${{ secrets.POSTGRES_PASS }}" `

Reply:
Oh, sorry, I understand now. It's confusing because you are using the same name the database container uses. The context is different. If we have another database vars, the more extensive name, it's clearer and easier to understand, and we must follow all other vars standards.